functorch/_src/eager_transforms.py

```diff
@@ -842,9 +846,9 @@ def push_jvp(basis):
             return jvp_out
 
         results = vmap(push_jvp)(basis)
-        assert_output_is_tensor_or_tensors(results, 'jacfwd(f, ...)(*args)')
-
         jac_outs, spec = tree_flatten(results)
+        assert_non_empty_output(jac_outs, 'jacfwd(f, ...)(*args)')
```

Comment:
Is this already covered by jvp?

Reply:
Yes. I'll remove it as well. I also spotted that in another PR.